hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/shell/Head.java

```diff
@@ -68,11 +70,9 @@ protected void processPath(PathData item) throws IOException {
   }
 
   private void dumpToOffset(PathData item) throws IOException {
-    FSDataInputStream in = item.fs.open(item.path);
-    try {
+    try (FSDataInputStream in = item.openFile(
+        FS_OPTION_OPENFILE_FADVISE_NORMAL)) {
```

Comment:
Isn't this the same as calling open(Path) as the code was doing previously?  Shouldn't it be sequential?